sermant-agentcore/sermant-agentcore-implement/src/main/java/io/sermant/implement/service/metrics/prometheus/PrometheusMeterRegistryProvider.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2024-2024 Sermant Authors. All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package io.sermant.implement.service.metrics.prometheus;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.prometheus.PrometheusConfig;
+import io.micrometer.prometheus.PrometheusMeterRegistry;
+import io.sermant.implement.service.metrics.MeterRegistryProvider;
+
+import org.kohsuke.MetaInfServices;
+
+/**
+ * Prometheus meter registry provider
+ *
+ * @author zwmagic
+ * @since 2024-08-19
+ */
+@MetaInfServices(MeterRegistryProvider.class)
```

Comment:
The spi configuration file is recommended.

Reply:
fixed